cypress/support/commands.js

```diff
@@ -99,6 +92,10 @@ Cypress.Commands.add('login', (username, password) => {
 
 Cypress.Commands.add('logout', () => {
   cy.visit('/user/logout');
+
+  // Handle confirmation dialog
+  cy.get('#edit-submit').click();
+
   cy.url().should('eq', Cypress.config().baseUrl + '/fi');
   // Check that authentication cookie doesn't exist anymore
   cy.getCookies().each((cookie) => {
```

Comment:
I would just click the logout link in the header, it has csrf token and the dialog is skipped all together.